usr/share/rear/restore/BORG/default/30_load_archives.sh

```diff
@@ -0,0 +1,62 @@
+# This file is part of Relax-and-Recover, licensed under the GNU General
+# Public License. Refer to the included COPYING for full text of license.
+#
+# 10_load_archives.sh
+
+LogPrint "Starting Borg restore"
+
+# Do we have Borg binary?
+# Missing Borg binary should not happen as it is copyied by
+# backup/BORG/default/20_start_backup.sh
+has_binary borg
+BugIfError "Could not find Borg binary"
+
+# Query Borg server for repoitory information and store it to archive_cache.
+# This should avoid repeatingly quering Borg server, which could be slow.
+archive_cache=$TMP_DIR/borg_archive
+borg list $BORG_USERNAME@$BORG_HOST:$BORG_REPO > $archive_cache
+StopIfError "Could not list Borg archive"
+
+# Store number of lines in archive_cache file for later use
+archive_cache_lines=$(wc -l $archive_cache | awk '{print $1}')
+
+# This means empty repository
+if [ $archive_cache_lines -eq 0 ]; then
+    Error "Borg repository $BORG_REPO on $BORG_HOST is empty"
+fi
```

Comment:
@ThomasWaldmann That is in OS restore stage. One you run restore and have nothing to restore from, there ain't much options left.